README.rst

```diff
@@ -1,6 +1,22 @@
 ftw.referencewidget
-==============
+===================
 
+This Widget provides a Referencebrowser and a searchfield, which allows users to select references.
+
+The basequery is all types which are not in the types_not_searched property all modification to the allowed types are relative to this query.
+
+Traversal or Selectability can be changed for all widgets with the IReferenceSettings registry interface or per widget with the widget parameters.
+
+The widget takes the following parameters:
+ - allow_traversal: These Types are traversable
+ - block_traversal: These Types are not traversable
+ - selectable: These Types are selectable
+ - nonselectable: These Types are not selectable
+ - start: The path first opened. Can either be a callable or a string
```

Comment:
Can I use situation based defaults here? 
For example:
- "here": use the object which I am currently editing as starting point (it's hard to hard code that path in the widget 😂 )
- "parent": use the parent object relative to "here"; useful for textblocks in content pages for example
- "navigation root": use the next navigation root as start point
- "site root": always start at the site root